ionic2/app/pages/poke-detail/poke-detail.page.html

```diff
@@ -26,8 +26,14 @@
         <hr>
       </ion-row>
       <ion-card>
-        <ion-card-content>
-        {{pokemon.description}}
+        <ion-card-content text-center>
+          {{pokemon.description}}
+        </ion-card-content>
+      </ion-card>
+      <ion-card>
+        <ion-card-content text-center>
+          {{(pokemon.appearanceLikelihood).toFixed(5)}}% of all Pokémon are a {{pokemon.name}}.<br/>
```

Comment:
What do you think displaying something like
`5 in every 1000 Pokemon appearances are a Bulbasaur.`
`<1 in every 1000 Pokemon appearances are a Mewtu.`
I think it's a lot easier to get an idea of the appearance likelihood compared to showing `0.00455%`.